WalletWasabi.Fluent/Models/Wallets/TransactionTreeBuilder.cs

```diff
@@ -174,6 +174,8 @@ private TransactionModel CreateCoinjoinGroup(int index, TransactionSummary trans
 			OrderIndex = index,
 			Type = TransactionType.CoinjoinGroup,
 			Status = status,
+			Fee = transactionSummary.GetFee(),
+			FeeRate = transactionSummary.FeeRate()
```

Comment:
Why do they need to be set here? This is a group, so everything depends on the children. So when `UpdateCoinjoinGroup()` method is called, that is the point when they should be set.

AFAIS `Fee` is actually calculated inside of that method, so setting it here is unnecessary, and I don't see that the FeeRate is used on ViewModel level so that might be unnecessary too for the group.

Reply:
Can I just remove and skip everything related to groups?

IMO it makes sense to show FeeRate for individual transactions, but if a CJ group got 7 tx and 5 of them is confirmed already, then what to show? 
FeeRate of the newest child? FeeRate of the oldest child? We can't make an average of them, because they are not correlated, only on the UI side it shown as a group.

Reply:
Remove them, yes.